fragments/arguments.sh

```diff
@@ -99,6 +100,16 @@ if [[ "$(whoami)" != "root" && "$DEBUG" -eq 0 ]]; then
     cleanupAndExit 6 "not running as root, exiting" ERROR
 fi
 
+
+# check Swift Dialog presence and version
+DIALOG_CMD="/usr/local/bin/dialog"
```

Comment:
Will this not override a custom value in the beginning of the script or set manually as an argument to the script call?

Reply:
This is `DIALOG_CMD` not `DIALOG_CMD_FILE`...